apps/docs/src/app/core/component-docs/input-group/examples/input-group-complex-example.component.html

```diff
@@ -1,19 +1,35 @@
 <div fd-form-item>
-    <label fd-form-label>Input Group with complex button addon.</label>
-    <fd-input-group>
-        <input fd-input-group-input fd-form-control/>
+    <label
+        for="input-group-complex-1"
+        id="fd-input-group-complex-label-1"
+        title="Input Group with complex button addon."
+        fd-form-label
+        >Input Group with complex button addon.</label
+    >
+    <fd-input-group id="input-group-complex-1">
+        <input aria-labelledby="fd-input-group-complex-label-1" fd-input-group-input fd-form-control />
         <span fd-input-group-addon button="true">
-            <button fd-button glyph="calendar" fdType="transparent"></button>
+            <button fd-button title="Calendar" glyph="calendar" fdType="transparent"></button>
         </span>
     </fd-input-group>
 </div>
 
-<br/>
+<br />
 
 <div fd-form-item>
-    <label fd-form-label>Input Group with complex span addon.</label>
-    <fd-input-group>
-        <input fd-input-group-input fd-form-control/>
-        <span fd-input-group-addon>Addon</span>
+    <label
+        for="input-group-complex-2"
+        id="fd-input-group-complex-label-2"
+        title="Input Group with complex span addon."
+        fd-form-label
+        >Input Group with complex span addon.</label
+    >
```

Comment:
I think, better formatting can be used here, like:
```
<label fd-form-label
    for="input-group-complex-2"
    id="fd-input-group-complex-label-2"
    title="Input Group with complex span addon."
>Input Group with complex span addon.</label>
```

Reply:
made changes as suggested